websites/D/dizipal/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "180731688844787712",
+		"name": "tuleyman"
+	},
+	"service": "dizipal",
+	"description": {
+		"en": "Dizipal current, Dizipal new and real address. Dizipal offers the latest TV series and movies safely and quickly.",
+		"tr": "Dizipal güncel, Dizipal yeni ve gerçek adres. Dizipal en güncel dizi ve filmleri güvenli ve hızlı bir şekilde sunar."
+	},
+	"url": "dizipal733.com",
+	"regExp": "dizipal[0-9]*[.]com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/CMzeCH3.png",
+	"thumbnail": "https://i.imgur.com/AgTzkyb.png",
+	"color": "#db2424",
+	"category": "videos",
+	"tags": [
+		"dizipal",
+		"tv-series",
+		"exen",
+		"netflix",
+		"dizipalcom",
+		"tb",
+		"tuleyman"
+	],
+	"iframe": true,
+	"matches": [
+		"*://dizipal[0-9]*.com/*"
+	],
+	"iFrameMatches": [
+		"*://dizipal[0-9]*.com/*"
```

Comment:
Unfortunately, these matches do not fit the [match pattern](https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns) requirements.
> If you use a wildcard in the host pattern, it must be the first or only character, and it must be followed by a period (.) or forward slash (/).